.github/workflows/--test.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         if: github.actor != 'dependabot[bot]' && steps.cache-coverage.outputs.cache-hit != 'true'
         id: docker
         with:
-          image: ghcr.io/myparcelnl/php-xd:7.2
+          image: ghcr.io/myparcelnl/php-xd:8.2
```

Comment:
Eigenlijk was de testsuite al niet geweldig want

A) We testen niet op alle PHP versies
B) We testen alleen op PHP 7.2 en niet op 7.1 welke volgens onze composer & README wel zou moeten werken

Hoe dan ook skippen we hier nu zoveel PHP versies dat het onmogelijk is om te weten of de boel in PHP < 8.2 nog werkt. Ik zou dit dan ook niet willen veranderen naar iets anders dan wat we in onze composer.json als versie meegeven (zie mijn comment daarbij)

Reply:
Het klopt dat die niet geweldig is, wat ook niet geweldig is is dat die api verzoeken doet met random data, waardoor sommige tests soms falen maar niet altijd. Ik denk niet dat ik in een story van 0 punten dat allemaal kan fixen, daarom koos ik voor een upgrade zodat alles weer werkte en de andere SDK stories door kunnen.